templates/smaily-woocommerce-admin.php

```diff
@@ -44,7 +46,18 @@
 		</ul>
 		</div>
 		<div class="message-display"></div>
-
+		<?php if ( $autoresponder_error ) : ?>
+		<div class="error-autoresponders error notice">
+			<p>
+			<?php
+				esc_html_e(
+					'There seems to be a problem with your connection to Smaily. Please revaidate your credentials!',
```

Comment:
You have typo in "Please revaidate your credentials!". It should be "Please revalidate your credentials!".

Reply:
Fixed in https://github.com/sendsmaily/smaily-woocommerce-plugin/pull/39/commits/20c8b71636360e81ee54db0622a777406774c1f9